views/scss/wizard.scss

```diff
@@ -0,0 +1,66 @@
+///
+/// This program is free software; you can redistribute it and/or
+/// modify it under the terms of the GNU General Public License
+/// as published by the Free Software Foundation; under version 2
+/// of the License (non-upgradable).
+///
+/// This program is distributed in the hope that it will be useful,
+/// but WITHOUT ANY WARRANTY; without even the implied warranty of
+/// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+/// GNU General Public License for more details.
+///
+/// You should have received a copy of the GNU General Public License
+/// along with this program; if not, write to the Free Software
+/// Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+/// Copyright (c) 2019 (original work) Open Assessment Technologies SA;
+///
+@import "inc/bootstrap";
+
+.section-container {
+    .content-block {
```

Comment:
is it necessary to have these two parent selectors? We just end up with unnecessarily long selectors in the output css.

Reply:
Necessary because the specificity should match the styles to be overridden in https://github.com/oat-sa/tao-core/blob/master/views/scss/inc/_section-container.scss#L206

Reply:
Ok